Build failed

hello, I am getting an error in this way on android,

```
> Task :react-native-azurenotificationhub:compileDebugJavaWithJavac FAILED
/Users/Y/Desktop/mobile/node_modules/react-native-azurenotificationhub/android/src/main/java/com/azure/reactnative/notificationhub/ReactNativeNotificationsHandler.java:117: error: constructor Builder in class Builder cannot be applied to given types;
            NotificationCompat.Builder notification = new NotificationCompat.Builder(context, NOTIFICATION_CHANNEL_ID)
```

react-native: 0.57.0

ext config

![screen shot 2018-10-03 at 12 32 43](https://user-images.githubusercontent.com/1306620/46402467-75833900-c708-11e8-8ce2-0a1d98b7ff3f.png)

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -24,9 +24,9 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.facebook.react:react-native:+'
-    compile 'com.google.android.gms:play-services-gcm:9.4.0'
-    compile 'com.google.firebase:firebase-messaging:9.4.0'
-    compile 'com.android.support:appcompat-v7:26.0.+'
+    compile 'com.google.android.gms:play-services-gcm:15.0.0'
+    compile 'com.google.firebase:firebase-messaging:15.0.0'
+    compile 'com.android.support:appcompat-v7:26.1.0'
     compile 'com.microsoft.azure:notification-hubs-android-sdk:0.4@aar'
     compile 'com.microsoft.azure:azure-notifications-handler:1.0.1@aar'
 }

